applovin_max/lib/applovin_max.dart

```diff
@@ -352,10 +352,15 @@ class AppLovinMAX {
   //
 
   /// Sets an [AdViewAdListener] listener with which you can receive notifications about ad events.
-  static void setBannerListener(AdViewAdListener listener) {
+  static void setBannerListener(AdViewAdListener? listener) {
     _bannerAdListener = listener;
   }
 
+  /// Unsets the [AdViewAdListener listener when it's no longer needed.
+  static void unsetBannerListener() {
```

Comment:
Is "unset" a Flutter term? I think I see "remove" listeners more often. Also, why do we need a new API as opposed to letting the user set a null listener in `setBannerListener(...)`?